uta_tools/data_sources/seqrepo_access.py

```diff
@@ -20,54 +20,35 @@ def __init__(self, seqrepo_data_path: str = SEQREPO_DATA_PATH) -> None:
         self.seqrepo_client = SeqRepo(seqrepo_data_path)
 
     def get_reference_sequence(
-            self, ac: str, start: int, end: Optional[int] = None,
+            self, ac: str, start: Optional[int] = None, end: Optional[int] = None,
             residue_mode: str = ResidueMode.RESIDUE
-    ) -> Tuple[Optional[str], Optional[str]]:
-        """Get reference sequence for transcript at given positions
+    ) -> Tuple[str, Optional[str]]:
+        """Get reference sequence for an accession given a start and end position.
+        If `start` and `end` are not given, it will return the entire reference sequence
```

Comment:
this also turns your terminal into the matrix for a minute which is a neat bonus feature